test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -249,6 +249,28 @@ describe("GET tests", () => {
     expect(resp.status).toEqual(500);
     expect(resp.text).toContain(SERVICE_UNAVAILABLE);
   });
+
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with identity check by uk agent`, async () => {
+    const applicationDataWithSomeoneElse = { ...APPLICATION_DATA_MOCK };
+    applicationDataWithSomeoneElse[WhoIsRegisteringKey] = WhoIsRegisteringType.AGENT;
+    mockGetApplicationData.mockReturnValueOnce(applicationDataWithSomeoneElse);
+
+    const resp = await request(app).get(CHECK_YOUR_ANSWERS_URL);
+
+    expect(resp.status).toEqual(200);
+    expect(resp.text).toContain("The UK agent");
```

Comment:
Could you please add text mock [here](https://github.com/companieshouse/overseas-entities-web/blob/main/test/__mocks__/text.mock.ts)

Reply:
Not sure what you mean.

Reply:
Clicked the link and can see what you mean. As these are only tested in one place I'm doubtful as to whether this is necessary.

Reply:
It is not just reusability but as well maintainability

Reply:
Not sure it really adds a lot in that way either what do other people @markpit think?

Reply:
I'd add it with the other test variables like Mo said to keep the code tidy.